package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.1.0",
   "description": "",
   "main": "index.js",
-  "bin": "index.js",
+  "bin": "./bin/krabby.js",
```

Comment:
`<yakshave>`
I don't think bins should have an extension, right?
`</yakshave>`